lib/src/extensions/widget_extension.dart

```diff
@@ -1384,4 +1384,32 @@ extension StyledWidget on Widget {
         maxHeight: maxHeight,
         child: this,
       );
+
+  Widget material({
+    Key? key,
+    MaterialType type = MaterialType.canvas,
+    double elevation = 0.0,
+    Color? color,
+    Color? shadowColor,
+    TextStyle? textStyle,
+    BorderRadiusGeometry? borderRadius,
+    ShapeBorder? shape,
+    bool borderOnForeground = true,
+    Clip clipBehavior = Clip.none,
+    Duration animationDuration = kThemeChangeDuration,
```

Comment:
Can you confirm this constant is the same as the default value of the Material widget?

Reply:
Sorry for the delay.

I do confirm that is it the default: https://api.flutter.dev/flutter/material/Material-class.html